parm/config/gfs/config.base

```diff
@@ -474,6 +474,14 @@ export DO_VRFY_OCEANDA="@DO_VRFY_OCEANDA@"  # Run SOCA Ocean and Seaice DA verif
 export FHMAX_FITS=132
 [[ "${FHMAX_FITS}" -gt "${FHMAX_GFS}" ]] && export FHMAX_FITS=${FHMAX_GFS}
 
+# Fetch options
+export DO_FETCH_HPSS="@DO_FETCH_HPSS@"   # pull from HPSS
+export DO_FETCH_LOCAL="@DO_FETCH_LOCAL@"   # pull from local
```

Comment:
These should probably just be `NO`.  If a user wants to fetch ICs, they will need to set this manually.  Maybe later, we can add an option to `setup_expt.py` that allows the user to specify a location/method to fetch ICs from, at which point templating these parameters will make sense.
```suggestion
export DO_FETCH_HPSS="NO"   # pull from HPSS
export DO_FETCH_LOCAL="NO"   # pull from local
```

Reply:
Ok, changed to "NO"